apps/rush-lib/src/cli/actions/TabCompleteAction.ts

```diff
@@ -0,0 +1,218 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { BaseRushAction } from './BaseRushAction';
+import { RushCommandLineParser } from '../RushCommandLineParser';
+
+import {
+  CommandLineStringParameter,
+  CommandLineIntegerParameter,
+  CommandLineParameterKind
+} from '@rushstack/ts-command-line';
+
+interface IParameter {
+  name: string;
+  kind: CommandLineParameterKind;
+}
+
+const DEFAULT_WORD_TO_AUTOCOMPLETE: string = '';
+const DEFAULT_POSITION: number = 0;
+export class TabCompleteAction extends BaseRushAction {
```

Comment:
It'd be cool if we could build this right into `ts-command-line`

Reply:
I thought about that, but chose to go this way for my FHL project - because it needs a better design. Let's do that in the future?

Reply:
Moved it to ts-command-line!